routes/api/v1.php

```diff
@@ -16,14 +16,16 @@
     UpdateStudentCollaborationsController,
     UpdateStudentProjectController,
     UpdateStudentSkillsController,
-    UpdateStudentProfileController};
+    UpdateStudentProfileController,
+    AddStudentLanguageController
+};
 
 use App\Http\Controllers\api\Tag\{
     TagListController,
     TagStoreController,
     TagDetailController,
     TagUpdateController,
-    DevelopmentListController
+    DevelopmentListController  
```

Comment:
There should be no blank spaces at the end

Reply:
done